pkg/scheduler/factory/factory.go

```diff
@@ -149,10 +149,10 @@ type Configurator interface {
 	GetPredicateMetadataProducer() (predicates.PredicateMetadataProducer, error)
 	GetPredicates(predicateKeys sets.String) (map[string]predicates.FitPredicate, error)
 
-	// Needs to be exposed for things like integration tests where we want to make fake nodes.
```

Comment:
why not squash the commits into the previous PR?

Reply:
I prefer to keep them separated.

Reply:
/hold

Feel free to cancel when the previous PR is merged.